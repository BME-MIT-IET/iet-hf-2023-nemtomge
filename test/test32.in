loadmap map.json
start V1 V2  
set Scientist V1 PL1 
give Gear V1 Gloves  
set Scientist V2 PL2  
move V1 PL2
give Crafted V2 Stun 3
give Crafted V2 Stun 3
give Crafted V2 Stun 3
use V2 V1  
1
use V2 V1  
1
use V2 V1  
1
has Gear V1 Gloves # should be false
exit
