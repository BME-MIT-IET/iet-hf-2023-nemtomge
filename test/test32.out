Map from file map.json loaded
The game has been started. Players are: V1 V2  
Scientist V1 set on field PL1 
Scientist V1 has been given gear Gloves 
Scientist V2 set on field PL2 
Scientist V1 moved to field PL2 
Scientist V2 has been given crafted agent Stun time: 3
Scientist V2 has been given crafted agent Stun time: 3
Scientist V2 has been given crafted agent Stun time: 3
Choose one to use: 
0 - cancel 
1 - Stun(3) 
2 - Stun(3) 
3 - Stun(3) 
Choose one to use: 
0 - cancel
1 - Stun(3) 
2 - Stun(3) 
Choose one to use: 
0 - cancel 
1 - Stun(3) 
Scientist V1 has gear Gloves uses: 0
